packages/mermaid/src/docs/intro/getting-started.md

```diff
@@ -1,27 +1,62 @@
-# A Mermaid User-Guide for Beginners
+---
+outline: 'deep' # shows all h3 headings in outline in Vitepress
+---
+
+# Mermaid User Guide
+
+## Mermaid is composed of three parts
+
+1. Deployment
+1. Syntax
+1. Configuration
+
+This section talks about the different ways to **deploy** Mermaid.
+
+If you are a beginner:
+
+- Check out the [Diagram Syntax](syntax-reference.md) page
+- Check out the [Tutorials](../config/Tutorials.md) page
+
+## Ways to use Mermaid
+
+1. [Using the Mermaid Chart Editor](getting-started.html#_1-using-the-mermaid-chart-editor)
+1. [Using the Mermaid Live Editor](getting-started.html#_1-using-the-mermaid-chart-editor)
+1. [Using Mermaid Plugins and Integrations](getting-started.html#_3-using-mermaid-plugins)
+1. [Calling the Mermaid JavaScript API](getting-started.html#_4-calling-the-mermaid-javascript-api)
+1. [Adding Mermaid as a dependency](getting-started.html#_5-adding-mermaid-as-a-dependency)
+
+To learn more, visit the [Usage](../config/usage.md) page.
+
+## 1. Using the Mermaid Chart Editor
+
+Available at the [Mermaid Chart](https://www.mermaidchart.com/) website.
 
-Mermaid is composed of three parts: Deployment, Syntax and Configuration.
+Mermaid Chart is a web-based diagram editor that allows you to create and edit diagrams in your browser. It is built by the team behind Mermaid.
 
-This section talks about the different ways to deploy Mermaid. Learning the [Syntax](syntax-reference.md) would be of great help to the beginner.
+Features include:
 
-> Generally the live editor is enough for most general uses of mermaid, and is a good place to start learning.
+- AI diagramming
+- Collaboration & multi-user editing
+- Storage
+- and more
 
-**Absolute beginners are advised to view the Video [Tutorials](../config/Tutorials.md) on the Live Editor, to gain a better understanding of mermaid.**
+To learn more, visit the [Mermaid Chart page](/ecosystem/mermaid-chart.html) in the Ecosystem section of the documentation.
 
-## Four ways of using mermaid:
+Or go to the [Mermaid Chart website](https://www.mermaidchart.com/app/sign-up) to sign up for a Free account.
 
-1. Using the Mermaid Live Editor at [mermaid.live](https://mermaid.live).
-2. Using [mermaid plugins](../ecosystem/integrations-community.md) with programs you are familiar with.
-3. Calling the Mermaid JavaScript API.
-4. Deploying Mermaid as a dependency.
+## 2. Using the Mermaid Live Editor
 
-**Note: It is our recommendation that you review all approaches, and choose the one that is best for your project.**
+Available at the [Mermaid Live Editor](https://mermaid.live) website.
 
-> More in depth information can be found at [Usage](../config/usage.md).
+### Features
 
-## 1. Using the Live Editor
+<br />
```

Comment:
I'm not a super big fan of these `<br />` (or adding any HTML to markdown files). It does make the generated website look nicer, though, so I see a reason for leaving them in.

Maybe I'm biased since I normally spend most of my time reading these docs in plain text :laughing: